pkgs/development/python-modules/vllm/default.nix

```diff
@@ -300,6 +300,11 @@ buildPythonPackage rec {
     ./0002-setup.py-nix-support-respect-cmakeFlags.patch
     ./0003-propagate-pythonpath.patch
     ./0005-drop-intel-reqs.patch
+    # Use system pynvml from nvidia-ml-py
+    (fetchpatch {
+      url = "https://github.com/daniel-fahey/vllm/commit/5cfd60530f57e461c4a1adc7d7d8e961b2ce1e4b.patch?full_index=1";
```

Comment:
Rather than use a patch here, can we do something more transparent with a substitute or something in the like ?
Adding a patch coming from a repo you control is not great. We shouldnt rely on trust is possible.